### ISSUE TITLE
Progress bars always showing 100%

### ISSUE THREAD

**Issue by dhersz** [ipeaGIT/geobr#154]
Progress bars "skips" from 0% to 100%, no matter how much of the action has been completed. I first noticed this when using `read_census_tract`, but it seems to extend to other functions, such as `read_municipality`, as well.

```r
sc <- read_census_tract(3304557)
#> Using year 2010
#>   |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%

rio <- read_municipality("RJ")
#> Using year 2010
#>   |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%
```

**Comment by rafapereirabr** [comment]
Thanks for the heads up. @CaioNG , could you please have a look at this?

**Comment by rafapereirabr** [comment]
This is a strange behavior from the `httr` package. [I've opend an issue in their repo](https://github.com/r-lib/httr/issues/683).

reprex.
```
library(httr)

file_url <- "http://www.ipea.gov.br/geobr/data_gpkg/census_tract/2010/33_simplified.gpkg"

httr::GET(url=file_url, httr::progress(type = "down") )

```

**Comment by rafapereirabr** [comment]
Fixed in dev version with #fd9fd3d7.
